tests/unit-ext/module/jerry-module-test.c

```diff
@@ -211,13 +212,18 @@ main (int argc, char **argv)
 
   js_function = jerry_create_external_function (handle_require);
   js_property_name = jerry_create_string ((const jerry_char_t *) "require");
-  jerry_set_property (js_global, js_property_name, js_function);
+  res = jerry_set_property (js_global, js_property_name, js_function);
+  TEST_ASSERT (!jerry_value_is_error (res));
+  TEST_ASSERT (jerry_get_boolean_value (res));
```

Comment:
Please add a `jerry_get_boolean_value` call too to ensure the result was a success. 
```c
TEST_ASSERT (jerry_get_boolean_value (res) && jerry_get_boolean_value (res));
```

Reply:
What is the purpose to call `jerry_get_boolean_value(res)` twice?
Did you actually mean `TEST_ASSERT (jerry_value_is_boolean (res) && jerry_get_boolean_value (res));`?

Reply:
@jerry-weng, Of course I did, sorry. Copy&paste error.

```
TEST_ASSERT (jerry_value_is_boolean (res) && jerry_get_boolean_value (res));
```